news/2025/2025-01-27-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,224 @@
+- A [fix](https://github.com/ppy/osu/pull/31525) proposed by [molneya](https://osu.ppy.sh/users/8945180) to ensure that the time between spinners are correctly accounted for in Flashlight calculations
+- A [fix](https://github.com/ppy/osu/pull/31447) proposed by [StanR](https://osu.ppy.sh/users/7217455) to ensure ODs below 0 cannot increase PP
+- A [fix](https://github.com/ppy/osu/pull/30544) proposed by [Finadoggie](https://osu.ppy.sh/users/14182048) to ensure estimated slider drops cannot go below zero
+- A [fix](https://github.com/ppy/osu/pull/30618) proposed by [Natelytle](https://osu.ppy.sh/users/17607667) to ensure osu!(lazer)'s PP counter does not display `NaN` while star rating is zero
+- A [set](https://github.com/ppy/osu/pull/30534) [of](https://github.com/ppy/osu/pull/30536) [refactors](https://github.com/ppy/osu/pull/31520) proposed by [StanR](https://osu.ppy.sh/users/7217455), [ltca](https://osu.ppy.sh/users/11475208) and [Natelytle](https://osu.ppy.sh/users/17607667) to aid with development
+- A [change](https://github.com/ppy/osu/pull/31449) proposed by [StanR](https://osu.ppy.sh/users/7217455) to simplify the angle bonus formulas in aim calculations
+- A [change](https://github.com/ppy/osu/pull/21211) proposed by [tsunyoku](https://osu.ppy.sh/users/11315329) to implement basic difficulty and performance calculations for the Autopilot mod
+
+## osu!taiko
+
+In order to aid understanding of the changes to osu!taiko, these are the skills in difficulty calculation which will be referenced throughout:
+
+- **Stamina**: the speed at which you hit notes, based on an assumed finger count of 2 per colour
+- **Colour**: the frequency or variability of which the beatmap changes between a don or kat
+- **Rhythm**: the complexity of the beatmap's rhythm in relation to notes' independent rhythm ratios
+
+Throughout these changes, we will be using the following terminology:
+
+- **Monos** refer to repeated notes of the same colour
+- **Intervals** refer to the snap interval of notes (such as 1/1, 1/4 and 1/8)
+
+### osu!taiko committee
+
+Since the last performance point post, there are some new faces in the osu!taiko pp committee:
+
+- [YaniFR](https://osu.ppy.sh/users/11260982)
+- [BabySnakes](https://osu.ppy.sh/users/4669728)
+- [rloseise](https://osu.ppy.sh/users/6793778)
+
+### Rhythm rewrite
+
+A [set](https://github.com/ppy/osu/pull/31284) [of](https://github.com/ppy/osu/pull/31339) [changes](https://github.com/ppy/osu/pull/31573) proposed by [ltca](https://osu.ppy.sh/users/11475208) (with contributions from [rloseise](https://osu.ppy.sh/users/6793778) and [YaniFR](https://osu.ppy.sh/users/11260982)) has been created in order to rewrite the rhythm skill to address shortcomings of the previous implementation.
```

Comment:
the past two updates included rhythm changes for osu! so i'd assume this is roughly the same idea. the explanation is similar

```suggestion
Similarly to how the osu! game mode has been changed in recent PP/SR updates to handle rhythm complexity, osu!taiko's rhythm calculations have been rewritten by a [set](https://github.com/ppy/osu/pull/31284) [of](https://github.com/ppy/osu/pull/31339) [changes](https://github.com/ppy/osu/pull/31573) proposed by [ltca](https://osu.ppy.sh/users/11475208) (with contributions from [rloseise](https://osu.ppy.sh/users/6793778) and [YaniFR](https://osu.ppy.sh/users/11260982)) to address shortcomings of the previous implementation.
```